flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/table/OracleTableSource.java

```diff
@@ -96,6 +100,9 @@ public OracleTableSource(
         this.startupOptions = startupOptions;
         this.producedDataType = physicalSchema.toPhysicalRowDataType();
         this.metadataKeys = Collections.emptyList();
+        checkArgument(
+                !(url == null && hostname == null),
+                "url and hostname can not be empty at the same time");
```

Comment:
We do not need validation here, if we want to validate in compile phase, we can add validation in OracleTableSourceFactory

Reply:
ok